### PR TITLE
[CU-2e70mqp] docs(book/developer-guides): changed banner image & added oracle set-up guide

### PR DESCRIPTION
**Pull request by Martin-Kiss**
## Issue
https://app.clickup.com/t/2e70mqp

## Description
- As discussed with Jafar changed the banner image of collator set-up-guide
- Added oracle set-up-guide
# Checklist

-[X] I have tested the guide with engineers and technical writers
- [ X] I have updated the `SUMMARY` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2e70mqp Please add Apollo oracle set-up guide to Mdbook](https://app.clickup.com/t/2e70mqp)

**Review by 0xBrainjar2 — DISMISSED** [review]

File: book/src/developer-guides/oracle-set-up-guide/oracle-set-up-guide.md [hunk 1]
```diff
@@ -0,0 +1,584 @@
...
+   <td>bottom drive obey lake curtain smoke basket hold race lonely fit walk//Alice
+   </td>
+  </tr>
+  <tr>
+   <td>publicKey
+   </td>
+   <td>5yNZjX24n2eg7W6EVamaTXNQbWCwchhThEaSWB7V3GRjtHeL
+   </td>
+  </tr>
+</table>
```
Comment (Dom-Roth) [inline book/src/developer-guides/oracle-set-up-guide/oracle-set-up-guide.md:470]: :point_up: HTML

File: book/src/developer-guides/collator-set-up-guide.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-![picasso-bannar](./picasso-bannar.png)
+![picasso_banner](./picasso-banner.png)
+
```
Comment (Dom-Roth) [inline book/src/developer-guides/collator-set-up-guide.md:2]: Space

File: book/src/developer-guides/oracle-set-up-guide/oracle-set-up-guide.md [hunk 3]
```diff
@@ -0,0 +1,584 @@
...
+
+
+**Create oracle for asset**
+
+We can create an oracle for an asset by using the extrinsic _oracle.addAssetAndInfo(...)_ and calling it using administrative rights. To call this extrinsic as an administrator go to:
+
+_Developer -> Sudo -> Oracle **-> addAssetAndInfo	**_
+
+
+    **Parameters**:
```
Comment (Dom-Roth) [inline book/src/developer-guides/oracle-set-up-guide/oracle-set-up-guide.md:553]: Tabbed?

**Review by Dom-Roth — DISMISSED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

File: book/src/developer-guides/oracle-set-up-guide/oracle-set-up-guide.md [hunk 4]
```diff
@@ -0,0 +1,484 @@
...
+
+It sets up a local network with 4 collators with predefined keys and addresses.
+
+**Setup required prerequisites**
+
+A **Debian** based Linux system is required, we recommend Debian, Ubuntu or Linux Mint.
+
+1. Set up required packages 
+
+Run the following command: \
```
Comment (benluelo) [inline book/src/developer-guides/oracle-set-up-guide/oracle-set-up-guide.md:40]:
```suggestion
Run the following command:
```

Comment (Martin-Kiss) [reply book/src/developer-guides/oracle-set-up-guide/oracle-set-up-guide.md:40]: noted

**Review by 0xBrainjar2 — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Pull request merged**